File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxCnt =0;
        int prevCnt = 0;
        for(int i=0;i<nums.length;i++) {
            if(nums[i] == 1) {
                prevCnt = prevCnt+1;
                maxCnt = Math.max(maxCnt, prevCnt);
            } else {
                prevCnt =0;
            }
        }
        return maxCnt;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        int maxCnt = 0;
        int prevCnt = 0;
        for(int i=0;i<nums.Length;i++) {
            if(nums[i] == 1) {
                prevCnt++;
                maxCnt = Math.Max(maxCnt,prevCnt);
            } else {
                prevCnt =0;
            }
        }
        return maxCnt;
    }
}